lib/src/json_schema/validator.dart

```diff
@@ -89,13 +108,15 @@ class Validator {
     }
   }
 
-  List<String> get errors => _errors.map((e) => e.toString()).toList();
+  ValidationResults _results;
 
-  List<String> get warnings => _warnings.map((e) => e.toString()).toList();
+  ValidationResults get results => _results;
```

Comment:
I wonder if we could just keep this private and return `_results` on a second and subsequent call to  `validateWithResults`?

Reply:
I suppose that's not realistic because we create a new Validator on each call. That's kind of lame. Maybe we should hold on to a validator privately once we create one?

Reply:
Oops, I don't think we need this.